roles/confluent.tools/tasks/main.yml

```diff
@@ -31,3 +31,11 @@
     owner: "{{kafka.tools.user}}"
     group: "{{kafka.tools.group}}"
   when: security_mode == "sasl_ssl"
+- name: tools ssl_customcerts config
+  template:
+    src: tools_ssl_customcerts.properties.j2
+    dest: "{{kafka.tools.config_file}}"
+    mode: 0640
+    owner: "{{kafka.tools.user}}"
```

Comment:
what does tools role even do? it doesn't start a service, just templates files... what uses those files??